TurkishFilter/sections/specific.txt

```diff
@@ -68,8 +68,7 @@ bursa.com,sozcu.com.tr,megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.
 /service/advertchannels?$xmlhttprequest,redirect=nooptext,domain=mardinhaber.com.tr|gidahatti.com|bolugundem.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
-||filmizletv.vip/Superbet$image
-||filmizletv.vip/suberbet$image
+/^https:\/\/filmizletv\.vip\/su(b|p)erbet/
```

Comment:
add domain resctriction

Reply:
`/^https:\/\/filmizletv\.vip\/`, a part of the regexp, does already restrict the domains that the regexp applies.